Core/src/IfcGeometryConverter/ProfileConverter.h

```diff
@@ -675,10 +665,10 @@ namespace OpenInfraPlatform {
 						if(asym_I_profile) {
 							if(asym_I_profile->TopFlangeWidth) {
 								double bTop = asym_I_profile->TopFlangeWidth * length_factor;																
-								double tfTop = asym_I_profile->TopFlangeThickness.value_or(tf);
-								double rTop = asym_I_profile->TopFlangeFilletRadius.value_or(r);
+								double tfTop = asym_I_profile->TopFlangeThickness.value_or(tf/ length_factor) * length_factor;
+								double rTop = asym_I_profile->TopFlangeFilletRadius.value_or(r/ length_factor) * length_factor;

```

Comment:
Use direct references to the default value - i.e. i_shape->FlangeThickness for tf.

Reply:
Done